swift-ticketing/src/swift_ticketing/model/ticket.clj

```diff
@@ -0,0 +1,38 @@
+(ns swift-ticketing.model.ticket
+  (:require [swift-ticketing.db.ticket :as db-ticket]
+            [swift-ticketing.db.booking :as db-booking]
+            [swift-ticketing.worker :as worker]))
+
+(defn create-tickets [db-spec uid event-id ticket-req]
+  (let [seat-type (:seat_type ticket-req)
+        tickets-map (if (= seat-type db-ticket/NAMED)
+                      (:seats ticket-req)
+                      (map (fn [_] {:name ""}) (range (:quantity ticket-req))))
+        price (:price ticket-req)
+        ticket-type-id (java.util.UUID/randomUUID)
+        tickets
+        (map (fn [m] (assoc m :ticket-id (java.util.UUID/randomUUID))) tickets-map)]
+    (db-ticket/insert-ticket-type db-spec event-id ticket-type-id ticket-req)
+    (db-ticket/insert-tickets db-spec ticket-type-id tickets price)
+    {:ticket-type-id ticket-type-id
+     :tickets tickets}))
+
+(defn reserve-ticket [db-spec message-queue uid event-id booking-req]
+  (let [booking-id (java.util.UUID/randomUUID)
+        req-ticket-ids (:ticket_ids booking-req)
+        ticket-ids (when-not (nil? req-ticket-ids)
+                     (map #(java.util.UUID/fromString %) req-ticket-ids))]
+    (db-booking/insert-booking db-spec uid booking-id db-booking/INPROCESS)
+    (worker/add-reserve-ticket-request-to-queue
+     message-queue
+     {:booking-id booking-id
+      :ticket-type-id (:ticket_type_id booking-req)
```

Comment:
I see a lot of snake-case keywords. The clojure convention is kebab-case. Can you convert these to kebab case? And JSON convention is camel-case. You can accept requests in camecase and convert them to kebab-case using coercion. Check if this makes sense: https://github.com/metosin/reitit/blob/master/doc/coercion/coercion.md?

Reply:
Before starting swift ticketing, I was going through [slack](https://api.slack.com/types/conversation) and [reddit](https://www.reddit.com/dev/api/) apis. They were using snake_case in json so I sticked to that. I'll switch to camelCase if that's the convention we use.
Will checkout coercion and convert all json keywords to kebab-case